dev-client/src/screens/SlopeScreen/SlopeScreen.tsx

```diff
@@ -68,14 +69,15 @@ export const SlopeScreen = ({siteId}: {siteId: string}) => {
       <DataInputSummary
```

Comment:
Issue: It appears the Figma design calls for a divider here too
```suggestion
      <Divider />
      <DataInputSummary
```